{"name": "test05_create_with_email_already_took", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at XCUITestDriver.doNativeFind (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:143:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at XCUITestDriver.findNativeElementOrElements (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:97:12)\n    at XCUITestDriver.findElOrEls (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:24:14)\n    at XCUITestDriver.findElOrElsWithProcessing (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:60:12)\n    at XCUITestDriver.findElement (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:75:12)", "trace": "self = <test_create_account.TestAccountCreation testMethod=test05_create_with_email_already_took>\n\n    @allure.story(\"Email already took\")\n    def test05_create_with_email_already_took(self) -> None:\n        new_account = get_new_account()\n        do.tap_join()\n        sleep(3)\n        do.input_email(test_account)\n        do.input_password(new_account.good_password)\n        do.tap_complete()\n>       do.assert_element(already_in_use, 'create failed by email already took')\n\ntestcase/account_creation/test_create_account.py:128: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/user_actions.py:776: in assert_element\n    raise e\nutils/user_actions.py:773: in assert_element\n    assert get_element(self.driver, locator, wait_time)\nutils/find_element.py:12: in get_element\n    raise e\nutils/find_element.py:6: in get_element\n    element = WebDriverWait(driver, wait_time).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9d48b836-86b3-4bde-9669-dd7e0fd8936a\")>\nmethod = <function get_element.<locals>.<lambda> at 0x105f6d430>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       NoSuchElementError: An element could not be located on the page using the given search parameters.\nE           at XCUITestDriver.doNativeFind (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:143:13)\nE           at processTicksAndRejections (node:internal/process/task_queues:95:5)\nE           at XCUITestDriver.findNativeElementOrElements (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:97:12)\nE           at XCUITestDriver.findElOrEls (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:24:14)\nE           at XCUITestDriver.findElOrElsWithProcessing (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:60:12)\nE           at XCUITestDriver.findElement (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:75:12)\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException"}, "start": 1690963189719, "stop": 1690963210023, "uuid": "e95b5d3b-907b-49e7-ad9b-dffacc1e1781", "historyId": "c8af7c38154f5cb81e41b97930b87c33", "testCaseId": "c8af7c38154f5cb81e41b97930b87c33", "fullName": "testcase.account_creation.test_create_account.TestAccountCreation#test05_create_with_email_already_took", "labels": [{"name": "feature", "value": "Account Creation"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "Email already took"}, {"name": "parentSuite", "value": "testcase.account_creation"}, {"name": "suite", "value": "test_create_account"}, {"name": "subSuite", "value": "TestAccountCreation"}, {"name": "host", "value": "Joys-MacBook-Pro.local"}, {"name": "thread", "value": "28553-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.account_creation.test_create_account"}]}